api/policy.go

```diff
@@ -240,6 +243,44 @@ func (p *EnvironmentTierPolicy) String() (string, error) {
 	return string(s), nil
 }
 
+// SensitiveDataPolicy is the policy configuration for sensitive data.
+// It is only applicable to database resource type.
+type SensitiveDataPolicy struct {
+	SensitiveDataList []SensitiveData `json:"sensitiveDataList"`
+}
+
+// SensitiveData is the value for sensitive data.
+type SensitiveData struct {
+	Table  string            `json:"table"`
+	Column string            `json:"column"`
+	Type   SensitiveDataType `json:"type"`
+}
+
+// SensitiveDataType is the type for sensitive data.
+type SensitiveDataType string
+
+const (
+	// SensitiveDataTypeWildcard is the sensitive data type for using wildcard to hide data.
+	SensitiveDataTypeWildcard SensitiveDataType = "WILDCARD"
```

Comment:
This is the mask type instead of the sensitive data type. So I would rename it to SensitiveDataMaskType. And define the first type as "DEFAULT", so that we will use the default masking for the corresponding data types.

Reply:
Updated in https://github.com/bytebase/bytebase/pull/3461